docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+
+#### New and changed features
+
+- Update package signing/verification has been enhanced.
+- Recovery package signer/verifier has been added.
+- Removable storage-based update/recovery has been supported. 
+- Update result API has been enhanced.
+- Diagnostics API, which can be used to retrieve diagnostic information from the system like system crash, has been added.
+- Ability to resolve callstack from updatable applications has been added.
+- Size-reduced coredumps for C# programs have been added.
+
+
+### System (Base)
+
+#### New and changed features
+
+- The capi-base-utils API set has been expanded.
+  - New
+    - ures: Resource management module.
+
+#### Fixes
+
+- The capi-system-system-settings API has been refactored.
+
+
+### Application framework
+
+#### New and changed features
+
+- TIDL Extension
+  - API set
+    - The API set for synchronous connection has been added.
+    - New keywords for sharing files have been added.
+  - Tools
+    - Tools for analyzing latency and throughput have been provided.
+    - The enhanced TIDL compiler using receive threads to improve concurrency has been provided.
+- Multi-package installer
+  - API set
+    - The API set has been provided to install multiple packages at once.
+  - Tools
+    - Tools have been provided to install multiple packages at once.
+- Tizen Theme Manager
+  - Getting resources
+    - The API set for obtaining related resources path using a specific resource key has been provided.
+  - Theme changed event
+    - The API set for receiving the changed event from the app when changing the settings has been provided.
+  - Theme overlay
+    - Overlay function between default theme and selected theme has been provided.
+
+
+### Window and Interaction
+
+#### New and changed features
+
+- Interaction
+  - Gesture FW & engine
+    - The API set to receive the event for hand gesture has been supported.
+    - Gesture Framework for managing multi gesture clients and engines has been supported.
+    - The service app for default gesture engine has been supported.
+    - Support for on-demand daemon launch has been added.
+- Enlightenment Wayland display server
+  - Display Compatibility for the Legacy Widget Application has been added. Legacy Widget applications can display on the screen of which size is different from the one of application without the modification.
+- Ecore Buffer
+  - The ecore-buffer and ecore-buffer-queue API set has been deprecated.
+- Tizen-ws-shell
+  - A new enumeration has been added to support a new type of quickpanel service.
+- Quickpanel C# Support
+  - C# API set for quickpanel client and service has been added. (Tizen.NUI.WindowSystem.Shell)
+- The libds API set that can be used to create and configure wayland-based display server has been newly added.
+
+
+### Graphics Engine
+
+#### New and changed features
+
+- DALi (3D UI Toolkit)
+  - All API set of DALi has been deprecated. NUI (DALi C# layer) replaces DALi API set.
```

Comment:
accept.

Reply:
As "API set" is being used instead of "APIs" in the release note, we will express as follows in order to avoid using "API sets".
  - API set of DALi has been deprecated. DALi API set has been replaced with NUI (DALi C# layer).